src/Quasar/Error.purs

```diff
@@ -49,6 +52,7 @@ printQError = case _ of
   Forbidden → "Resource is unavailable, the current authorization credentials do not grant access to the resource"
   PaymentRequired → "Resource is unavailable, payment is required to use this feature"
   ErrorMessage {title, message} → maybe "" (_ <> ": ") title <> message
+  MultipleErrors errs → "Multiple errors occured: " <> intercalate ", " (map printQError errs)
```

Comment:
I didn't know `intercalate` worked for Strings in PS! I've been using `Str.joinWith` all this time...